pyproject.toml

```diff
@@ -1,3 +1,26 @@
+[tool.poetry]
```

Comment:
there is a bit of meta information missing here that's previously been included in the package:
- classifiers
- license
- maintainers
- hompage
- repository
- documentation
- keywords
- include (files from MANIFEST.in) 

Also need to check if there are any files that previously got added to the package by `include_package_data=True` in `setup.py`, they now need to be explicitly listed in the `include` section.